app/models/user.py

```diff
@@ -62,6 +62,7 @@ class User(UserMixin, db.Model):
     password_hash = db.Column(db.String(128))
     role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))
     agency_id = db.Column(db.Integer, db.ForeignKey('agencies.id'))
+    reported_incidents = db.Relationship("IdlingIncident")
```

Comment:
Can we make the relationship in the IdlingIncident model just to be consistent with how we made other relationships to User?